app/serializers/admin_user_list_serializer.rb

```diff
@@ -122,6 +122,10 @@ def include_can_be_deleted?
     @options[:include_can_be_deleted]
   end
 
+  def silence_reason
+    PrettyText.cleanup(object.silence_reason)
```

Comment:
It looks like `object.silence_reason` is given by `UserHistory#details` which can have a maximum length of 110_000 characters and parsing with Nokogiri can have a significant impact on performance. I'm wondering if there is a reason we can't trust `UserHistory#details` as already sanitized when we store it in the database.

Reply:
> I'm wondering if there is a reason we can't trust `UserHistory#details` as already sanitized when we store it in the database.

I had a look through the code and it turns out that we don't sanitize the message even storing it in the database even though the `message` and `reason` params is user input.

Reply:
After some discussion, we realized that this is hard to untangle so we will accept the performance risk until it proves to be an actual problem.